src/client/src/services/EngineAPIService.tsx

```diff
@@ -12,41 +12,6 @@ export default class EngineAPIService {
     }
   }
```

Comment:
This entire file should be removed. If we haven't implemented a way to get the pages from the extension we should do that first.